indico/modules/events/registration/badges.py

```diff
@@ -83,7 +83,8 @@ def _draw_badge(self, canvas, registration, template, tpl_data, pos_x, pos_y):
 
         # Print images first
         image_placeholders = {name for name, placeholder in placeholders.items() if placeholder.is_image}
-        items = sorted(tpl_data.items, key=lambda item: item['type'] not in image_placeholders)
+        items = sorted(tpl_data.items, key=lambda item: (int(item.get('zIndex', 65535)),
```

Comment:
Not sure about 65535 as a default value. Do we want "legacy" items to show up in front by default? Wouldn't 0 be a better choice? And maybe a comment to explain the reasoning would be of help.

Reply:
I don't have a reason for choosing a higher value but we can't use 0 because that would place the item behind the background image.

Reply:
Yeah, well, 10, in this case.